schema205/json_translate.py

```diff
@@ -96,16 +106,57 @@ def add_data_group(self, group_name, group_subdict):
             if 'Notes' in element:
                 elements['properties'][e]['notes'] = element['Notes']
             if 'Required' in element:
-                required.append(e)
+                #required.append(e)
+                req = element['Required']
+                if isinstance(req, bool) and req == True:
+                    required.append(e)
+                elif req.startswith('if'):
```

Comment:
The JSON output for RS0002 is showing:

```json
"if": {
    "properties": {
        "staging_type": {
            "not": {
                "const": "SINGLE_STAGE"
            }
        }
    }
},
"then": {
    "required": [
        "cooling_low_air_volumetric_flow_rate"
    ]
},
```

`cooling_low_air_volumetric_flow_rate` is the last element that fits this criteria, but many others should be listed.

Reply:
I fixed an unresolved TODO - the requirements list should be correct in the latest push.